ui/packages/catalog-ui-search/src/main/webapp/react-component/dropdown/dropdown.js

```diff
@@ -206,4 +203,20 @@ class Dropdown extends React.Component {
   }
 }
 
+const Dropdown = ({ label, anchor, ...props }) => {
+  const dropdownProps = {
```

Comment:
Is this necessary? You could conditionally render within DropdownBody as well.

Reply:
I think this is fine either way - adding it outside the Dropdown body should help clarify that this isn't actually part of the body, and is just the part of the dropdown that you click to open it.